htpy/__init__.py

```diff
@@ -74,12 +76,18 @@ def _id_class_names_from_css_str(x: t.Any) -> dict[str, Attribute]:
     return result
 
 
-def _kwarg_attribute_name(name: str) -> str:
+def _html_name(name: str) -> str:
```

Comment:
```suggestion
def _python_to_html_name(name: str) -> str:
```

suggestion: maybe it is clearer?